odoo/modules/module.py

```diff
@@ -504,7 +505,7 @@ def run_unit_tests(module_name, dbname, position=runs_at_install):
     r = True
     for m in mods:
         tests = unwrap_suite(unittest.TestLoader().loadTestsFromModule(m))
-        suite = unittest.TestSuite(t for t in tests if position(t))
+        suite = unittest.TestSuite(t for t in tests if position(t) and check_tags(t, tools.config['test_tags']))
```

Comment:
Maybe the tags-parsing could be done just once outside the loop rather than for every test?

Maybe even in the post-processing steps of the commands/config parsing? I think we have stuff like that.